examples/VideoClassification/VideoClassification.ipynb

```diff
@@ -0,0 +1,359 @@
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "=========================SUMMARY=========================\n",
+      "mmvu_eval_32images-fb15e25a compared to mmvu_eval_32images-04ae25b8:\n",
+      "45.00% 'LLMJudge' score\n",
+      "\n",
+      "1739836712.14s start\n",
+      "1739836729.72s end\n",
+      "15.47s duration\n",
+      "8.23s llm_duration\n",
+      "2472.25tok prompt_tokens\n",
+      "32.40tok completion_tokens\n",
+      "2504.65tok total_tokens\n",
+      "0.01$ estimated_cost\n",
+      "\n",
+      "See results for mmvu_eval_32images-fb15e25a at https://www.braintrust.dev/app/braintrustdata.com/p/mmvu_eval_32images/experiments/mmvu_eval_32images-fb15e25a\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "EvalResultWithSummary(summary=\"...\", results=[...])"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import os\n",
+    "import base64\n",
+    "from typing import List, Dict, Any, Optional\n",
+    "import asyncio\n",
+    "\n",
+    "import cv2\n",
+    "import requests\n",
+    "from requests.adapters import HTTPAdapter\n",
+    "from urllib3.util.retry import Retry\n",
+    "\n",
+    "from datasets import load_dataset\n",
+    "\n",
+    "import braintrust\n",
+    "import autoevals\n",
+    "from openai import OpenAI\n",
+    "\n",
+    "\n",
+    "NUM_FRAMES = 32\n",
+    "TARGET_DIMENSIONS = (512, 512)\n",
+    "JPEG_QUALITY = 80\n",
+    "\n",
+    "RETRY_TOTAL = 3\n",
+    "RETRY_BACKOFF = 0.5\n",
+    "STATUS_FORCELIST = [502, 503, 504]\n",
+    "\n",
+    "\n",
+    "os.environ[\"BRAINTRUST_API_KEY\"] = \"API_KEY_HERE\"\n",
+    "\n",
+    "\n",
+    "client = braintrust.wrap_openai(\n",
+    "    OpenAI(\n",
+    "        api_key=os.environ[\"BRAINTRUST_API_KEY\"],\n",
+    "        base_url=\"https://api.braintrust.dev/v1/proxy\",\n",
+    "    )\n",
+    ")\n",
+    "\n",
+    "\n",
+    "def extract_frames_b64(video_path: str) -> List[str]:\n",
+    "    video_capture = cv2.VideoCapture(video_path)\n",
+    "    base64_frames = []\n",
+    "    count = 0\n",
+    "\n",
+    "    while video_capture.isOpened() and count < NUM_FRAMES:\n",
+    "        ret, frame = video_capture.read()\n",
+    "        if not ret:\n",
+    "            break\n",
+    "\n",
+    "        frame = cv2.resize(frame, TARGET_DIMENSIONS)\n",
+    "        success, encoded_img = cv2.imencode(\n",
+    "            \".jpg\", frame, [int(cv2.IMWRITE_JPEG_QUALITY), JPEG_QUALITY]\n",
+    "        )\n",
+    "        if success:\n",
+    "            b64_str = base64.b64encode(encoded_img).decode(\"utf-8\")\n",
+    "            base64_frames.append(b64_str)\n",
+    "        count += 1\n",
+    "\n",
+    "    video_capture.release()\n",
```

Comment:
The best way to do this would be a `try/finally` block, to guarantee that we always release it. So something like
```
video_capture = cv2.VideoCapture(...)
try:
    ... # All of the code except for `video_capture.release()`
finally:
    video_capture.release()
```